pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -983,7 +983,7 @@ class Build {
                     throw new Exception()
                 }
             } finally {
-                if (buildConfig.TARGET_OS == "aix") {
+                if (buildConfig.TARGET_OS == "aix" || buildConfig.ARCHITECTURE == "s390x") {
```

Comment:
We should add a comment as to why we need to do this.

Reply:
added comment